RetrospectiveExtension.Frontend/.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "default": false
```

Comment:
Disabling the default markdown lint rules to limit PR to CI related work. Will enable in future PR with required markdown changes.

Reply:
Yes - will enable and fix in future PR.